.gitignore

```diff
@@ -78,3 +78,4 @@ engine/src/main/resources/entity_store.dat
 
 # Ignore donwloaded JREs
 jre/**
+Log.log.txt
```

Comment:
Huh, this is an odd one. Where did this file come from, or did this ignore entry get added by accident?

Reply:
I removed the file name now it will be fine